sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/implementation/CryptographyClientImpl.java

```diff
@@ -102,15 +100,7 @@ public Response<KeyVaultKey> getKey(Context context) {
         return new SimpleResponse<>(response, createKeyVaultKey(response.getValue()));
     }
 
-    public Mono<JsonWebKey> getSecretKeyAsync() {
```

Comment:
Both types of clients only need `getSecretKey*` operations it to create a local crypto client. In this version of the PR we're using the sync method to instantiate the client, as we only need to call it once and I figured it's be easier to maintain just one version of that method rather than to write both a sync and async versions. Since we would call this before executing any async crypto operations (encrypt, decrypt, wrap, etc.) I figured it was ok to use the sync get as it's not blocking within the async call. Do you think it'd be better tomake the whole thing async?

Reply:
I added this back.